### PR TITLE
fix uninitialized variable

### PR DESCRIPTION
**Pull request by rodolforg**
Coverity issue CID 308861

### CODE REVIEW

File: synfig-core/src/synfig/color/cairocolor.h [hunk 1]
```diff
@@ -192,13 +192,14 @@ class CairoColor
...
-	CairoColor(const CairoColor& c, const unsigned char A):
-	pixel(c.get_pixel()) { set_a(A); }
+	CairoColor(const unsigned char R, const unsigned char G, const unsigned char B, const unsigned char A=ceil)
+		: pixel((A<<24)|(R<<16)|(G<<8)|(B)) { }
+	CairoColor(const CairoColor& c, const unsigned char A)
+		: pixel(c.get_pixel()) { set_a(A); }
 
 	// Conversor constructor
 	CairoColor(const Color& c)
+		: pixel(0)
```
Comment (ice0) [inline synfig-core/src/synfig/color/cairocolor.h:202]: Seems it should be initialized with c.get_pixel()?

File: synfig-core/src/synfig/color/cairocolor.h [hunk 2]
```diff
@@ -192,13 +192,14 @@ class CairoColor
...
-	CairoColor(const unsigned char R, const unsigned char G, const unsigned char B, const unsigned char A=ceil):
-	pixel((A<<24)|(R<<16)|(G<<8)|(B)) { }
-	CairoColor(const CairoColor& c, const unsigned char A):
-	pixel(c.get_pixel()) { set_a(A); }
+	CairoColor(const unsigned char R, const unsigned char G, const unsigned char B, const unsigned char A=ceil)
+		: pixel((A<<24)|(R<<16)|(G<<8)|(B)) { }
+	CairoColor(const CairoColor& c, const unsigned char A)
+		: pixel(c.get_pixel()) { set_a(A); }
 
 	// Conversor constructor
```
Comment (ice0) [inline synfig-core/src/synfig/color/cairocolor.h:200]: Converter? constructor

Comment (rodolforg) [reply synfig-core/src/synfig/color/cairocolor.h]: fixed that comment typo

File: synfig-core/src/synfig/color/cairocolor.h (hunk 1 above)
Comment (rodolforg) [reply synfig-core/src/synfig/color/cairocolor.h:202]: synfig::Color doesn't have `get_pixel()`, only `CairoColor`. By the way it's never used outside CairColor class, so I'll delete it.

Comment (ice0) [reply synfig-core/src/synfig/color/cairocolor.h:202]: Ok.

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**